src/hevm/test/test.hs

```diff
@@ -398,33 +398,7 @@ tests = testGroup "hevm"
           [Cex _, Cex _] <- withSolvers Z3 1 $ \s -> checkAssert s defaultPanicCodes c (Just ("fun(uint256)", [AbiUIntType 256, AbiUIntType 256])) []
           putStrLn "expected 2 counterexamples found"
         ,
-        expectFail $ testCase "Deposit contract loop (z3)" $ do
```

Comment:
NOTE: I am removing this because it's a copy-paste of the one below, which has now been fixed, except that this had everything in the code commented out.